website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx

```diff
@@ -0,0 +1,16 @@
+### Transit Sign API calls with managed keys fail
+
+#### Affected versions
+
+- 1.14.0+ up to 1.14.3 inclusively.
```

Comment:
```suggestion
- 1.14.0+ up to 1.14.3 inclusively
```

Style correction: format similar page elements consistently

Reply:
Applied in https://github.com/hashicorp/vault/pull/23080/commits/73a206f089239be8eee21737a8db955c6bbe762b